src/main/java/properties/EnvironmentConfig.java

```diff
@@ -0,0 +1,86 @@
+package properties;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.Properties;
+
+/***
+ * Project environment data
+ */
+public class EnvironmentConfig {
+
+    private static final String MAIN_URL = "url";
+    private static final String REGISTRATION_URL = "regURl";
+    private static final String CATALOGUE_URL = "catalogueURL";
+    private static final String CART_URL = "cartURL";
+    private static final String PASSWORD = "password";
+    private static final String LOGIN = "login";
+
+    private final static String ENVIRONMENT_PROPERTIES_PATH = "src/main/resources/application-%s.properties";
+    private final static String LOCAL_PROPERTIES_PATH = "src/main/resources/application-local.properties";
```

Comment:
Why do you have a separate variable for local environment file path? Why don't you get file name from pattern ENVIRONMENT_PROPERTIES_PATH?

Reply:
fixed